packages/accessible-react-app/src/css/layout.css

```diff
@@ -48,14 +49,14 @@ img {
   justify-content: space-between;
 }
 
-@media (min-width: 30em) {
+@media (min-width: 768px) {
```

Comment:
What if we used rems instead of px? IE 11 support there https://caniuse.com/?search=rem it would be 48rem (786/16) assuming 16 is the base font size